docs/multisite/languages/languages.md

```diff
@@ -22,15 +22,15 @@ However, even if work on a draft takes time and other translations are updated i
 ### Adding available languages
 
 The multilanguage system operates based on a global translation list that contains all languages available in the installation.
-Languages can be [added to this list from the **Admin** panel](https://doc.ibexa.co/projects/userguide/en/latest/content_management/translate_content/) in the back office.
+Languages can be [added to this list from the **Admin** panel]([[= user_doc =]]/content_management/translate_content/) in the back office.
 After adding a language be sure to dump all assets to the file system:
 
 ```
 yarn encore <environment>
 # OR php bin/console ibexa:encore:compile
 ```
 
-**The new language must then be added to the [SiteAccess](multisite.md) configuration**.
+**The new language must then be added to the [SiteAccess](multisite.md) configuration, both for the frontend and back office SiteAccesses.**
```

Comment:
In our docs we have equal numbers of "frontends" "front ends" and "front-ends".
We might want to decide on one, wdys?